go-controller/pkg/ovn/ovn.go

```diff
@@ -72,6 +88,11 @@ type Controller struct {
 
 	// supports port_group?
 	portGroupSupport bool
+
+	// Map of load balancers to service namespace
+	serviceVIPToName map[ServiceVIPKey]types.NamespacedName
+
+	serviceVIPToNameLock *sync.Mutex
```

Comment:
I don't think this needs to be a `*`, it can just be a plain `sync.Mutex`

Reply:
Done